src/main/java/tqs/example/impostor/controller/AdminController.java

```diff
@@ -0,0 +1,39 @@
+package tqs.example.impostor.controller;
+
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.view.RedirectView;
+import tqs.example.impostor.service.AdminService;
+
+@RestController
+@RequestMapping("/admin")
+public class AdminController {
```

Comment:
I'm not sure if the AdminController class is considered finished or not, but if it is, then shouldn't there be a couple more endpoints here?